Source/Fuse.Reactive.Expressions/NameValuePair.uno

```diff
@@ -0,0 +1,21 @@
+using Uno.UX;
+
+namespace Fuse.Reactive
+{
+	/** Creates a `Fuse.NameValuePair` from a name and a value. */
+	public class NameValuePair: UnaryOperator
+	{
+		public string Name { get; private set; }
+
+		[UXConstructor]
+		public NameValuePair([UXParameter("Name")] string name, [UXParameter("Value")] Expression value) : base(value)
+		{
+			Name = name;
+		}
+
+		protected override object Compute(object value)
+		{
+			return new Fuse.NameValuePair(Name, value);
```

Comment:
We could, but I figured it would be a rarely used feature and it adds quite a lot of cost. We could add this later if it turns out to be in high demand.